### PR TITLE
Cleanups + Fix escaping error

### LINKED ISSUE #108: VSCode shows empty page after update
__Describe the Bug__

I created a diagram via your extension and saved. When I close and reopen the file in bpm.io editor everything is visible.

However when I made changes to parts of the process, save and close the diagram, I get a blank screen in vscode bpmn.io extension rather than my (updated) diagram.

If I open the latest revision on the website https://demo.bpmn.io, my process is clearly visible and editable.

more info: if you add `<bpmn2:text>` to your diagram and they contain a single quote `'`, the diagram will no longer be visible via vscode.

__Steps to Reproduce__

1. Open old version of the attached process in vscode extension => diagram is visible
2. Open new version of the attached process in vscode extension => white page
3. Open the new version of the process and remove the quote on position 178:53 ==> diagram is visible again

__Expected Behavior__

I would like to see the diagram in your extension and not have to go to demo.bpmn.io

__Environment__

- Host (Browser/Node version), if applicable: VSCode 1.70.1 with BPMN.io Editor extension v0.13.0 (updated on 11 april 2022)
- OS: Windows 10 21H2
- Library version: N/A

[old-new-bpmnfile.zip](https://github.com/bpmn-io/vs-code-bpmn-io/files/9673281/old-new-bpmnfile.zip)

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #108 

---

**Housekeeping** 🧹 
* migrate to `npm@8`
* bump dev dependencies
* bump to `bpmn-js@10`

Additionally, this fixes one bug reported via #108. I found that the VS Code Webview has problems when importing `'` characters inside text annotations. `moddle-xml` itself has no problems with that, so I fixed that by simply escaping these chars before import.

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Follow-up:
* Integrate renovate
* Release a new version ✨

**Review by barmac — APPROVED** [review]

File: src/features/editing/bpmnModelerBuilder.ts [hunk 1]
```diff
@@ -12,8 +12,12 @@ export class BpmnModelerBuilder {
     return contents.replace(/(\r\n|\n|\r)/gm, ' ');
   }
 
+  private replaceSingleQuotes(contents: string): string {
+    return contents.replace(/'/gm, "&#39;");
```
Comment (nikku) [inline src/features/editing/bpmnModelerBuilder.ts:16]: What is the issue here? Is it VSCode or us messing up?

Comment (nikku) [reply src/features/editing/bpmnModelerBuilder.ts:16]: Like I'm wondering if this is valid XML; If so we shall fix this in moddle not this library.

Comment (nikku) [reply src/features/editing/bpmnModelerBuilder.ts:16]: Works in Camunda Desktop Modeler: 

image

Comment (pinussilvestrus) [reply src/features/editing/bpmnModelerBuilder.ts:16]: It is a valid XML, therefore I think that shouldn't be fixed in moddle. I tried a few roundtrips between the extension, our bpmn-io demo, and the Desktop Modeler.

VS Code messes the single quotes up. As we currently go the simple way and provide [all webview content as a single string](https://github.com/bpmn-io/vs-code-bpmn-io/blob/master/src/features/editing/bpmnModelerBuilder.ts#L53), it simply breaks once the content string has another `'` in between.

**Pull request merged**